crypto.go

```diff
@@ -183,14 +183,22 @@ func keyToCertificate(sk ic.PrivKey) (*tls.Certificate, error) {
 		return nil, err
 	}
 
-	sn, err := rand.Int(rand.Reader, big.NewInt(1<<62))
+	bigNum := big.NewInt(1 << 62)
+	sn, err := rand.Int(rand.Reader, bigNum)
+	if err != nil {
+		return nil, err
+	}
+	issuerSN, err := rand.Int(rand.Reader, bigNum)
 	if err != nil {
 		return nil, err
 	}
 	tmpl := &x509.Certificate{
 		SerialNumber: sn,
 		NotBefore:    time.Time{},
 		NotAfter:     time.Now().Add(certValidityPeriod),
+		// According to RFC 3280, the issuer field must be set,
+		// see https://datatracker.ietf.org/doc/html/rfc3280#section-4.1.2.4.
+		Issuer: pkix.Name{SerialNumber: issuerSN.String()},
```

Comment:
`x509.CreateCertificate` will populate the `Issuer` field automatically using `Subject` from the parent template.
```suggestion
		Subject: pkix.Name{SerialNumber: issuerSN.String()},
```